src/Evolution/DgSubcell/PerssonTci.cpp

```diff
@@ -23,27 +23,61 @@ namespace evolution::dg::subcell::detail {
 template <size_t Dim>
```

Comment:
Commit message: "Use pow instead of log in PerssonTci" and then in the details part of the message briefly explain why and that the zero cutoff isn't needed. The current algorithm isn't broken, it's exactly what the original paper describes :)